### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by tolaked**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+require('dotenv').config();
+const express = require('express');
+
+const connectDB = require('./database');
+const resources = require('./resources');
```
Comment (ayenisholah) [inline app.js:5]: Comment out line 5 and any other WIP code that crashes the server.

File: app.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+const expressMiddlewares = require('./utils/middlewares');
+
+const app = express();
+
+
+expressMiddlewares(app);
+
+app.use(resources);
```
Comment (ayenisholah) [inline app.js:13]: Line 13 also crashes the server.

File: app.js (hunk 1 above)
Comment (tolaked) [reply app.js:5]: Thanks for the feedback, well noted.

**Comment by ayenisholah** [comment]
Great job.

**Pull request merged**